Sources/Sugar/Validators/StrongPassword.swift

```diff
@@ -0,0 +1,24 @@
+import Vapor
+
+private let regex = "^.*(?=.{3,})(?=.*[a-zA-Z])(?=.*[0-9])(?=.*[\\d\\X])(?=.*[!$#%]).*$"
+/*The password contains characters from at least three of the following five categories:
+
+ English uppercase characters (A – Z)
+ English lowercase characters (a – z)
+ Base 10 digits (0 – 9)
+ Non-alphanumeric (For example: !, $, #, or %)
+ Unicode characters*/
+
+public struct StrongPassword {
+    public init() {}
+
+    public func validate(_ input: String) throws {
+        guard input.range(of: regex, options: .regularExpression) != nil else {
```

Comment:
how about allowing the StrongPassword to be initialized several requirements like min/max password length.
Also, StrongPassword does not conform to Validator yet, correct?